olive/platform_sdk/qualcomm/env.py

```diff
@@ -11,7 +11,7 @@
 
 
 class SDKEnv:
-    def __init__(self, sdk: str, root_env_name: str, target_arch: str = None, dev: bool = False):
+    def __init__(self, sdk: str, root_env_name: str, target_arch: str = None, optional_local_run: bool = False):
```

Comment:
I think a name like `use_dev_tools` is a better name than `optional_local_run`. Even an inference can be run locally.

Reply:
Updated.